PyViCare/PyViCareService.py

```diff
@@ -238,12 +244,12 @@ def get(self, url):
 
    #TODO should move to device after refactoring 
     def getProperty(self,property_name):
-        url = apiURLBase + '/operational-data/installations/' + str(self.id) + '/gateways/' + str(self.serial) + '/devices/0/features/' + property_name
+        url = buildGetPropertyUrl(self.id, self.serial, property_name)
```

Comment:
Why did you get rid of the 404 error handling?

Reply:
Good side is that you added that anyway. I'll merge and test.

Reply:
What do you mean with 404 handling? I just extracted the URL concatenation, so I can use the same logic inside the service mock.